runtime/jcl/common/sun_misc_Perf.c

```diff
@@ -125,7 +126,11 @@ registerJdkInternalPerfPerfNatives(JNIEnv *env, jclass clazz) {
 			(void *)&Java_sun_misc_Perf_highResFrequency
 		},
 	};
-	(*env)->RegisterNatives(env, clazz, natives, sizeof(natives)/sizeof(JNINativeMethod));
+	jint numNatives = sizeof(natives)/sizeof(JNINativeMethod);
```

Comment:
Pls declare c variables at the beginning of a scope.

Reply:
Should have looked at more context, this is the beginning.

Reply:
Same here.